imgee/utils.py

```diff
@@ -97,7 +97,7 @@ def get_file_url(scheme=None):
 
 
 def path_for(img_name):
-    return os.path.join(app.config['UPLOADED_FILES_DEST'], img_name)
+    return os.path.join(app.project_root, app.config['UPLOADED_FILES_DEST'], img_name)
```

Comment:
Don't use this. If files are hosted under the static folder, use `app.static_folder`